README.md

```diff
@@ -1,75 +1,49 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Project Name 
+To do list.
 
-> Description the project.
+# Description of project.
+This To-do list App is built using webpack and served by a webpack dev server
 
+## Project web demo link. 
 
-## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+Built With Html. Css. JS.
 
-## Live Demo (if available)
+Website link: 
 
-[Live Demo Link](https://livedemo.com)
+# Getting starting 
+To get a local copy up and running follow these simple example steps:
 
+## Prerequisites
+ Vscode. 
+ must have git installed.
 
-## Getting Started
+## Setup 
+Clone this repo with:
+https://github.com/eerapheal/toDolist
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+cd awesomebook-ES6
 
+## Install 
+npm install
 
-To get a local copy up and running follow these simple example steps.
+Author1 Ekpenisi Erue Raphael.
 
-### Prerequisites
+GitHub: (https://github.com/eerapheal); 
```

Comment:
- [ ] OPTIONAL Here you can use [link name](link URL) structure to display the links more pretty. Example [GitHub](https://github.com/eerapheal) here I used this `[GitHub](https://github.com/eerapheal)`